virtualmachine/virtualmachine.go

```diff
@@ -22,6 +22,8 @@ type VirtualMachine interface {
 	Halt() error
 	Start() error
 	GetSSH(ssh.Options) (ssh.Client, error)
+	AddDisk() error
+	RemoveDisk() error
```

Comment:
can't change the interface, this will break every other virtualmachine in libretto.